deploy/nebula/templates/clusterdefinition.yaml

```diff
@@ -0,0 +1,199 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ClusterDefinition
+metadata:
+  name: nebula
+  labels:
+    {{- include "nebula.labels" . | nindent 4 }}
+spec:
+  connectionCredential:
+    username: root
+    password: "$(RANDOM_PASSWD)"
+    endpoint: "$(SVC_FQDN):$(SVC_PORT_tcp-nebula)"
+    host: "$(SVC_FQDN)"
+    port: "$(SVC_PORT_tcp-nebula)"
+    accesskey: ""
+    secretkey: ""
+  componentDefs:
```

Comment:
following components missing telemetry configs?

Reply:
Prefered to support monitoring-related functions in future PRs. Added an issue here: https://github.com/apecloud/kubeblocks/issues/3863.

Reply:
Also remember to add connect(a DAPR binding is needed), backup & restore(in tools) in near future